Editor/NRepl.cs

```diff
@@ -511,6 +477,20 @@ static void HandleMessage (BDictionary message, TcpClient client)
                             {"session", session.ToString()},
                             {"status", new BList {"done"}},
 							{"completions", completions}
+                        }, client);
+					break;
+				case "classpath":
+                    // Debug.Log("Classpath op has been called");
+					string assetsPath = Path.Combine(Path.GetFullPath("."), "Assets");
+					string arcadiaSrcPath = Path.Combine(assetsPath, "Arcadia", "Source");
```

Comment:
Instead of hardcoding the Arcadia and Source folders, you should be able to get the computed load path from             

```cls
Environment.GetEnvironmentVariable("CLOJURE_LOAD_PATH");
```

We populate that during initialization and its the value ClojureCLR uses internally when its searching for files. It's the closest thing we have to a class path and it takes into consideration any leiningen projects you might have under Assets as well. It is a string you can split using `System.IO.Path.PathSeparator`.

Reply:
Done! I had to add an extra check, because (at least on my machine) there was an empty path added to `CLOJURE_LOAD_PATH`. So what we send back now is the list of paths in `CLOJURE_LOAD_PATH` after removing empty strings.

Reply:
Great!

> there was an empty path added to `CLOJURE_LOAD_PATH`

Ah interesting. I wonder if that's a bug on our end. I will look into it.